tv/api/queries/getSessions/index.ts

```diff
@@ -140,11 +140,15 @@ const formattedSession = function (
     session.speaker = sessionResult.speakers.results[0].name;
   }
 
-  session.Day = day.taxonomyName;
-  session.ShortDay = day.sortOrder;
+  if (day != null) {
```

Comment:
Not an issue, but a suggestion for next time you change JavaScript code. In JS, we normally use `===` and `!==` comparison operators that also validate types. To compare against null, it does not matter though.

A shorthand often used by JS developers is `if (day) { ... }`. This checks if `day` is truthy (has a value other than null, undefined, 0, or empty string.